test/test_roo.rb

```diff
@@ -26,14 +26,15 @@ class TestRoo < Test::Unit::TestCase
   GOOGLE       = false 	# do Google-Spreadsheet Tests?
   EXCELX       = true  	# do Excelx Tests? (.xlsx files)
   LIBREOFFICE  = true  	# do LibreOffice tests? (.ods files)
-  CSV          = true  	# do CSV tests? (.csv files)
+  CSV          = false  	# do CSV tests? (.csv files)

```

Comment:
Some of the tests assume that there are multiple sheets (which CSV doesn't support) so it breaks a lot of tests. 

There was a test for the each_row method but it only validated against XLS. Since the change was local to CSV.each_row I added code to validate each_row for CSV but not have CSV run for all tests since it doesn't make sense to run multiple sheet tests against CSV files. Which is what happens when you set the above property to true.